addons/web_editor/static/src/js/wysiwyg/wysiwyg.js

```diff
@@ -1042,17 +1048,24 @@ const Wysiwyg = Widget.extend({
             this._updateFaResizeButtons();
         }
         const link = getInSelection(this.odooEditor.document, 'a');
-        if (isInMedia || link && !this.options.preventLinkDoubleClick) {
-            // Handle the media/link's tooltip.
-            this.showTooltip = true;
-            setTimeout(() => {
-                // Do not show tooltip on double-click and if there is already one
-                if (!this.showTooltip || $target.attr('title') !== undefined) {
-                    return;
-                }
-                $target.tooltip({title: _t('Double-click to edit'), trigger: 'manual', container: 'body'}).tooltip('show');
-                setTimeout(() => $target.tooltip('dispose'), 800);
-            }, 400);
+        if (isInMedia || link) {
+            if (!this.options.preventLinkDoubleClick) {
+                // Handle the media/link's tooltip.
+                this.showTooltip = true;
+                setTimeout(() => {
+                    // Do not show tooltip on double-click and if there is already one
+                    if (!this.showTooltip || $target.attr('title') !== undefined) {
+                        return;
+                    }
+                    $target.tooltip({title: _t('Double-click to edit'), trigger: 'manual', container: 'body'}).tooltip('show');
+                    setTimeout(() => $target.tooltip('dispose'), 800);
+                }, 400);
+            } else {
+                this.toggleLinkTools({
```

Comment:
To check but isn't removing this line just enough? https://github.com/odoo-dev/odoo/commit/86cf231bf1a4c80f4c7bccdae406287ac90d9916#diff-92e15a4998e507e1b666dc06a4673f7a686d687b32d5eef193646be7fd80d3faR53

And then maybe remove every occurence of `preventLinkDoubleClick` as I implemented it in the editor just to prevent dbl click from website as the spec wanted.. but now we revert that

In the start method of this widget, you will see dbl click already toggleLinkTools if the options is not enable